lib/active_merchant/billing/gateways/reach.rb

```diff
@@ -56,6 +56,40 @@ def capture(money, authorization, options = {})
         commit('capture', post)
       end
 
+      def supports_scrubbing?
```

Comment:
💬  Comment:
 This PR should not affect the #supports_scrubbing? and #scrub methods.

Reply:
Done!